packages/typescript-sdk/package.json

```diff
@@ -28,6 +28,9 @@
     "dist"
   ],
   "scripts": {
+    "organize_imports": "find src/generated -type f -name '*.ts' | xargs organize-imports-cli",
```

Comment:
Interesting that you have to do so much shell magic to organize imports 😄 I would expect that this is a usual task and would be better supported by `organize-imports-cli` 😞

Reply:
Actually whenever i searched for organizing imports it was UI instructions, and the typescript linter removed it, so i was stuck with this 😅 . not perfect but it does the job 👍